service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java

```diff
@@ -38,6 +38,14 @@ public boolean isCtscReportEnabled() {
             false);
     }
 
+    public boolean isSDONotificationForAllocatedJudgeEnabled() {
+        return ldClient.boolVariation("allocatedJudgeNotificationSDO", ldUser, false);
+    }
+
+    public boolean isNoticeOfProceedingsNotificationForAllocatedJudgeEnabled() {
+        return ldClient.boolVariation("allocatedJudgeNotificationNoticeOfProceedings", ldUser, false);
+    }
```

Comment:
is it worth making these static to remove readability?

Reply:
not sure about this one, the rest of the toggles in this file follow same format as above. what do you think? @Jamesnelson117

Reply:
Yeah was thinking that, keep it as is for now